package.json

```diff
@@ -50,8 +50,8 @@
     "lint:fix": "tsc --noEmit && eslint \"**/*.{js,ts}\" --quiet --fix",
     "prettier": "prettier --check \"**/*.{js,ts}\"",
     "prettier:fix": "prettier --write \"**/*.{js,ts}\"",
-    "test": "jest --silent",
-    "test:coverage": "jest --collectCoverage --silent",
+    "test": "jest",
+    "test:coverage": "jest --collectCoverage",
```

Comment:
Just double checking this was an intentional change...

Reply:
haha yep it was, should've done this a while ago when we removed console logs from the source code